201-azure-relay-create-all-resources/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
-    "itemDisplayName": "Create an Azure Relay namespace with all resources",
-    "description": "This template enables you to deploy an Azure Relay namespace with standard SKU, a WCF Relay and a HybridConnection.",
-    "summary": "This template creates an Azure Relay namespace, a WCF Relay and a HybridConnection.",
-    "githubUsername": "grandhiramesh",
-    "dateUpdated": "2016-10-25"
+  "itemDisplayName": "Create an Azure Relay namespace with all resources",
+  "description": "This template enables you to deploy an Azure Relay namespace with standard SKU, a WCF Relay and a HybridConnection.",
+  "summary": "This template creates an Azure Relay namespace, a WCF Relay and a HybridConnection.",
+  "githubUsername": "v-ajnava",
```

Comment:
do you want to take ownership of all of these samples?

Reply:
Yes, talked with Team here